src/components/Explore/utils.ts

```diff
@@ -20,36 +19,86 @@ import regions, {
   USA,
 } from 'common/regions';
 import { fail } from 'assert';
-import { pluralize } from 'common/utils';
+import { pluralize, formatPercent, formatDecimal } from 'common/utils';
 import {
   TimeUnit,
   DateFormat,
   formatDateTime,
   getTimeDiff,
 } from 'common/utils/time-utils';
-import { formatDecimal } from 'common/utils/index';
+import { subtractTime } from 'common/utils/time-utils';
 
-export function getMaxBy<T>(
-  seriesList: Series[],
-  getValue: (d: Column) => T,
-  defaultValue: T,
-): T {
-  const maxValue = max(seriesList.map(({ data }) => max(data.map(getValue))));
-  return maxValue || defaultValue;
+export enum Period {
+  DAYS_60,
+  DAYS_180,
+  ALL,
 }
 
-export enum ExploreMetric {
-  CASES,
-  DEATHS,
-  HOSPITALIZATIONS,
-  ICU_HOSPITALIZATIONS,
+const EXPLORE_PERIODS = [Period.DAYS_60, Period.DAYS_180, Period.ALL];
+
+interface PeriodDefinition {
+  increment: number;
+  label: string;
+  xTicktimeUnit: TimeUnit;
+}
+
+export const periodMap: {
+  [period in Period]: PeriodDefinition;
+} = {
+  [Period.DAYS_60]: {
+    increment: 60,
+    label: '60',
+    xTicktimeUnit: TimeUnit.WEEKS,
```

Comment:
sets the x axis ticks to render in increments of weeks for the 60-day period (which was previously just showing 1 tick- 'May'- on mobile). [implemented here](https://github.com/covid-projections/covid-projections/pull/3670/files#diff-94fef4efde3155913fc63fdabe97e9081a90d10b28fd2107de31826e0f239f5fR181)